pack/side/tmg_encounter.json

```diff
@@ -0,0 +1,1421 @@
+    "code": "71001",
+    "double_sided": true,
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "name": "The Midwinter Gala",
+    "pack_code": "tmg",
+    "position": 1,
+    "quantity": 1,
+    "text": "Easy / Standard\n[skull]: -X. X is the current agenda number.\n[cultist]: -X. X is the number of [[Guest]] assets at your location (to a maximum of 5).\n[tablet]: -2. (-4 instead if you are at a [[Private]] location).\n[elder_thing]: -3. If you fail or if The Bloodless Man is at your location, take 1 horror.",
+    "type_code": "scenario"
+  },
+  {
+    "back_flavor": "A crash resounds from somewhere deeper in the manor, followed by a scream. Declan Pearce emerges to calm the crowd in the ballroom, but you can't help but notice the masked attendants have positioned themselves in front of each of the manor's many exits.",
+    "back_name": "Who Let the Gugs Out?",
+    "back_text": "Shuffle each set-aside [[Monster]] enemy, both copies of the Vicious Ambush treachery, and the encounter discard pile into the encounter deck.",
+    "code": "71002",
+    "doom": 4,
+    "double_sided": true,
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "Promising to be a night of intrigue and wonder, the gala is expected to be filled with secrets and whispers that are as much a part of the entertainment as the drinks and music.",
+    "illustrator": "Robert Laskey",
+    "name": "Masked Revelers",
+    "pack_code": "tmg",
+    "position": 2,
+    "quantity": 1,
+    "stage": 1,
+    "text": "Each [[Lantern Club]] enemy cannot be dealt damage.\nDoom cannot be removed from this agenda by card effects.\n<b>Forced</b> - When a single-sided [[Guest]] asset would leave play: It becomes spellbound instead <i>(flip it facedown, lose control of it, and heal all damage and horror from it)</i>.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "Mayhem Ensues",
+    "back_text": "<b>If The Bloodless Man (Silent Lurker) is in play:</b>\n<blockquote><i>You turn around to see the Bloodless Man striding toward you with murderous intent. Before you can react, the imposing figure staggers, then falls face first onto the red carpet.<br><br>Moments later, a sharp sound of ripping cloth breaks the silence. Glistening appendages erupt from the Bloodless Man's torn suit and limbs, blossoming into a grotesque amalgam of flesh and writhing tentacles. Your stomach churns as the creature's bone-white mask stares down at you, set in a maw of nightmarish limbs and gnashing fangs.</i></blockquote>\nFlip The Bloodless Man <i>(Silent Lurker)</i> to his <i>(Unleashed)</i> side.\n<hr>\n<b>Otherwise:</b>\n<blockquote><i>The clamor around you seems to quiet for a moment.</i></blockquote>\nEach investigator may heal 1 horror from an investigator or [[Guest]] asset at their location.",
```

Comment:
```suggestion
    "back_text": "<b>If The Bloodless Man <i>(Silent Lurker)</i> is in play:</b>\n<blockquote><i>You turn around to see the Bloodless Man striding toward you with murderous intent. Before you can react, the imposing figure staggers, then falls face first onto the red carpet.<br><br>Moments later, a sharp sound of ripping cloth breaks the silence. Glistening appendages erupt from the Bloodless Man's torn suit and limbs, blossoming into a grotesque amalgam of flesh and writhing tentacles. Your stomach churns as the creature's bone-white mask stares down at you, set in a maw of nightmarish limbs and gnashing fangs.</i></blockquote>\nFlip The Bloodless Man <i>(Silent Lurker)</i> to his <i>(Unleashed)</i> side.\n<hr>\n<b>Otherwise:</b>\n<blockquote><i>The clamor around you seems to quiet for a moment.</i></blockquote>\nEach investigator may heal 1 horror from an investigator or [[Guest]] asset at their location.",
```